readme.md

```diff
@@ -4,7 +4,7 @@
 **Requires at least:** 4.9.9  
 **Requires PHP:** 5.6  
 **Tested up to:** 5.7  
-**Stable tag:** 3.6.2  
+**Stable tag:** 3.7.0  
```

Comment:
If `expires = on` is the default behaviour, I would recommend to use 4.0.0 as version, because it is a breaking change.

Reply:
Agreed. Also will add the option to restore the original behavior just in case